src/calypso-utils.js

```diff
@@ -0,0 +1,32 @@
+"use strict";
+const docblock = require('jest-docblock');
+
+/**
+ * Returns true if the given text contains @format.
+ * within its first docblock. False otherwise.
+ *
+ * @param {String} text text to scan for the format keyword within the first docblock
+ */
+const shouldFormat = text => {
+  const directives = docblock.parse(text);
+  return Object.keys(directives).indexOf('format') >= 0;
```

Comment:
it didn't work that way but i'm forgetting the error message now. 

i should have made a todo to come back around and fully understand what was going on there.